cpp/models/ode_secir/parameters_io.h

```diff
@@ -70,20 +71,19 @@ IOResult<void> read_confirmed_cases_data(
     const std::vector<double>& scaling_factor_inf);
 
 /**
-     * @brief sets populations data from json file with multiple age groups into a Model with one age group
-     * @tparam FP floating point data type, e.g., double
-     * @param[in, out] model vector of objects in which the data is set
-     * @param[in] path Path to confirmed cases file
-     * @param[in] region vector of keys of the region of interest
-     * @param[in] year Specifies year at which the data is read
-     * @param[in] month Specifies month at which the data is read
-     * @param[in] day Specifies day at which the data is read
-     * @param[in] scaling_factor_inf factors by which to scale the confirmed cases of rki data
-     */
+ * @brief sets populations data from already read data with multiple age groups into a Model with one age group
```

Comment:
Maybe you can refactor the documentation and start each sentence with a capital letter and end it with a dot.